docker-compose.yml

```diff
@@ -0,0 +1,98 @@
+version: "3.8"
+services:
+  # RPC daemon
+  bitcoin:
+    image: ghcr.io/vulpemventures/bitcoin:latest
+    user: 1000:1000
+    container_name: bitcoin
+    command:
+      - -regtest=1
+      - -port=18444
+      - -rpcport=18443
+      - -txindex=1
+      - -rpcuser=admin1
+      - -rpcpassword=123
+      - -rpcallowip=0.0.0.0/0
+      - -rpcbind=0.0.0.0
+      - -fallbackfee=0.1
+    ports:
+      - 18443:18443
+      - 18444:18444
+      # ZMQ
+      - 28332:28332
+      - 28333:28333
+    volumes:
+      - bitcoin:/config
+    restart: unless-stopped
+
+  # Block explorer server
+  electrs:
+    image: ghcr.io/vulpemventures/electrs:latest
+    container_name: electrs
+    entrypoint:
+      - /build/electrs
+    command:
+      - -vvvv
+      - --network
+      - regtest
+      - --daemon-dir
+      - /config
+      - --daemon-rpc-addr
+      - bitcoin:18443
+      - --cookie
+      - admin1:123
+      - --http-addr
+      - 0.0.0.0:30000
+      - --electrum-rpc-addr
+      - 0.0.0.0:50000
+      - --cors
+      - "*"
+      - --jsonrpc-import
+    depends_on:
+      - bitcoin
+    ports:
+      - 50000:50000
+      - 30000:30000
+    volumes:
+      - bitcoin:/config
+    restart: unless-stopped
+
+  # Block explorer frontend
+  esplora:
```

Comment:
🔧 We don't have to start a blockchain explorer for the tests.

Reply:
I'll leave it in this file so that we can use it locally (instead of nigiri)